django/contrib/staticfiles/views.py

```diff
@@ -27,6 +29,12 @@ def serve(request, path, insecure=False, **kwargs):
 
     It uses the django.views.static.serve() view to serve the found files.
     """
+    warnings.warn(
+        "django.contrib.staticfiles.views.serve is deprecated",
```

Comment:
`django.views.static.serve()` is used to serve media files during development and by  `django.test.testcase.LiveServerTestCase` for media files and static files during tests.
WhiteNoise can only serve static files. 
See: https://groups.google.com/d/msg/django-developers/N0KbgDeLuUE/el_xYOaaDwAJ
 
`django.contrib.staticfiles.StaticLiveServerTestCase` was created to make LiveServerTestCase and with it Django Core not depend on the staticfiles app, so I didn't integrate WhiteNoise with LiveServerTestCase.
See: https://code.djangoproject.com/ticket/20739